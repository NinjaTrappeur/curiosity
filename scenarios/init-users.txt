# Initialize a state with four users.
user signup alice a alice@example.com --accept-tos
user update USER-1 Alice "Hi ! I'm the first advisor registered on this prototype. I know and have access to everything... Let me know if you need anything !"

user signup bob b bob@example.com --accept-tos
user signup charlie c charlie@example.com --accept-tos

user signup mila m mila@example.com --accept-tos
user update USER-4 Mila "I'm Mila, the first member to join this prototype. My advisor is @alice and I've created @alpha."

user signup chloe c chloe@example.com --accept-tos
