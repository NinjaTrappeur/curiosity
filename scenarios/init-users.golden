2: user signup alice a alice@example.com --accept-tos
User created: USER-1
3: user update USER-1 Alice Hi ! I'm the first advisor registered on this prototype. I know and have access to everything... Let me know if you need anything !
User updated: USER-1
5: user signup bob b bob@example.com --accept-tos
User created: USER-2
6: user signup charlie c charlie@example.com --accept-tos
User created: USER-3
8: user signup mila m mila@example.com --accept-tos
User created: USER-4
9: user update USER-4 Mila I'm Mila, the first member to join this prototype. My advisor is @alice and I've created @alpha.
User updated: USER-4
11: user signup chloe c chloe@example.com --accept-tos
User created: USER-5
